packages/perseus-core/src/widgets/interactive-graph/interactive-graph-util.test.ts

```diff
@@ -0,0 +1,40 @@
+import getInteractiveGraphPublicWidgetOptions from "./interactive-graph-util";
+
+import type {PerseusInteractiveGraphWidgetOptions} from "@khanacademy/perseus-core";
+
+describe("getInteractiveGraphPublicWidgetOptions", () => {
+    it("removes the `correct` field", () => {
+        const options: PerseusInteractiveGraphWidgetOptions = {
+            correct: {
+                type: "angle",
```

Comment:
Sorry, I'm late to this PR... 

Should we have one more test here for `static: true`?  When that's the case, we need to keep the `correct` field as that's used in the render: https://github.com/Khan/perseus/blob/9cabe689a7aa143f95adf4556bf5c10d654a66ae/packages/perseus/src/widgets/interactive-graphs/stateful-mafs-graph.tsx#L136

Reply:
The static field doesn't exist on `PerseusInteractiveGraphWidgetOptions`, only `WidgetOptions`, so we'll have to do this check elsewhere as in my comment above. The static field on `WidgetOptions` is passed to each widget by the renderer: https://github.com/Khan/perseus/blob/b07f2936fccb94cde5cd34ce258973581611416a/packages/perseus/src/renderer.tsx#L588.

Reply:
🤦‍♂️  Yikes, sorry! I totally missed the comments above relating to this already! I switched straight to the file changes and didn't see them.